drivers/clock_control/clock_fixed_rate.c

```diff
@@ -0,0 +1,79 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright (C) 2022 Google, LLC
+ *
+ */
+
+#include <zephyr/drivers/clock_control.h>
+
+#define DT_DRV_COMPAT fixed_clock
+
+struct fixed_rate_clock_config {
+	uint32_t rate;
+};
+
+static int fixed_rate_clk_on(const struct device *dev,
+			     clock_control_subsys_t sys)
+{
+	ARG_UNUSED(dev);
```

Comment:
If `fixed_rate_clk_init` is empty and you aren't doing anything on `fixed_rate_clk_on`, where is the clock actually enabled on the SoC ?

Reply:
It's an always-on clock (e.g. external), it doesn't get enabled, or it might have gotten enabled by a previous boot loader.